client/app/components/ApplicationArea/ApplicationLayout/DesktopNavbar.jsx

```diff
@@ -84,23 +84,23 @@ export default function DesktopNavbar() {
         {currentUser.hasPermission("list_dashboards") && (
           <Menu.Item key="dashboards" className={activeState.dashboards ? "navbar-active-item" : null}>
             <Link href="dashboards">
-              <DesktopOutlinedIcon />
+              <DesktopOutlinedIcon alt="Dashboard navigation button" />
               <span className="desktop-navbar-label">Dashboards</span>
             </Link>
           </Menu.Item>
         )}
         {currentUser.hasPermission("view_query") && (
           <Menu.Item key="queries" className={activeState.queries ? "navbar-active-item" : null}>
             <Link href="queries">
-              <CodeOutlinedIcon />
+              <CodeOutlinedIcon alt="Queries navigation button" />
```

Comment:
Are these giving you warnings when you use the accessibility tool? `alt` is just an attribute for `img` elements (even though some browsers play nicely with it). In general, you want `aria-label`. `ant` appears to use the `img` role, not an `img` tag, for its icons. 


(This comment applies to all `alt` icon attributes)

Reply:
(Reference: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Role_Img#svg_and_roleimg)